dist/selleckt-browserify.js

```diff
@@ -0,0 +1,1154 @@
+
+        this.items = itemsObj.items;
+
+        if(itemsObj.selectedItems[0]){
+            this.selectItem(itemsObj.selectedItems[0], { silent: true });
+        }
+    },
+
+    findItemInList: function(item){
+        return this.$sellecktEl.find('.'+this.itemClass+'[data-value="' + item.value + '"]');
+    },
+
+    findItem: function(value){
+        return _(this.items).find(function(item){
+            /*jshint eqeqeq:false*/
+            return item.value == value;
+        });
+    },
+
+    hideSelectionFromChoices: function(){
+        if(!this.$sellecktEl){
+            return;
+        }
+
+        if(this.selectedItem){
+            this.findItemInList(this.selectedItem).hide();
+        }
+    },
+
+    selectItem: function(item, options){
+        var selectedItem = this.selectedItem,
+            $sellecktEl = this.$sellecktEl,
+            displayedLabel = (item && item.label) || this.placeholderText;
+
+        item = item || {};
+        options = options || {};
+
+        if(selectedItem){
+            this.findItemInList(selectedItem).show();
+        }
+
+        if($sellecktEl){
+            this.findItemInList(item).hide();
+            $sellecktEl.find('.'+this.selectedTextClass).text(displayedLabel);
+        }
+
+        this.selectedItem = item;
+
+        if (!options.silent) {
+            this.$originalSelectEl.val(item.value).trigger('change', {origin: 'selleckt'});
+        }
+
+        this.hideSelectionFromChoices();
+
+        this.trigger('itemSelected', item);
+    },
+
+    updateSelection: function(newSelection){
+        this.selectItem(this.findItem(newSelection));
+    },
+
+    getSelection: function(){
+        return this.selectedItem;
+    },
+
+    getTemplateData: function() {
+        var selectedItem = this.getSelection();
+
+        return _.extend(this.mainTemplateData, {
+            showSearch : this.showSearch,
+            selectedItemText: selectedItem && selectedItem.label || this.placeholderText,
+            className : this.className,
+            selectedClass: this.selectedClass,
+            selectedTextClass: this.selectedTextClass,
+            itemsClass: this.itemsClass,
+            itemslistClass : this.itemslistClass,
+            itemClass: this.itemClass,
+            itemTextClass: this.itemTextClass,
+            searchInputClass: this.searchInputClass,
+            items: this.items
+        });
+    },
+
+    addItems: function(items){
+        var selectItem = _.bind(this.selectItem, this);
+        var $options = _.map(items, function(item){
+            return this._createOptionFromItem(item);
+        }, this);
+
+        this.$originalSelectEl.append($options);
+
+        this.items = this.items.concat(items);
+
+        var selectedItem = _.find(items, function(item){
+            return item.isSelected;
+        });
+
+        if(!selectedItem){
+            return;
+        }
+
+        _.delay(function(){
```

Comment:
This will require all of the tests which use `addItems` to wait `DELAY_TIMEOUT` before making their assertions...